handbook/marketing/content/web-series.md

```diff
@@ -0,0 +1,71 @@
+## Web series
+
+Describe what a web series is and why we do them at Sourcegraph
+
+## Dev Tool Time
+
+Dev Tool Time is a bi-weekly web series livestreamed on Twitch where Sourcegraph engineers interview other devs in the community about their desk set up, favorite dev tools, productivity hacks, and more.
+
+This series aims to help new developers learn about existing tools but also update experienced developers who might not know about the latest developer tools. While Sourcegraph may be mentioned, it will not be the main focus. This series is genuinely to inform developers about all the different tools out there and enhance developer awareness about the importance of having the right tools setup to increase coding efficiency.
```

Comment:
I'd personally remove the "latest", because showing off new things is not a focus, IMHO.

```suggestion
This series aims to help new developers learn about existing tools but also update experienced developers who might not know about certain developer tools and workflows. While Sourcegraph may be mentioned, it will not be the main focus. This series is genuinely to inform developers about all the different tools out there and enhance developer awareness about the importance of having the right tools setup to increase coding efficiency.
```